Sprint-1/exercises/paths.js

```diff
@@ -14,5 +14,11 @@ const lastSlashIndex = filePath.lastIndexOf("/");
 const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
 
+
+
 // Create a variable to store the dir part of the filePath variable
+const filePathDir = filePath.slice(- 41);
```

Comment:
Your code should work for any valid file path, not just the sample value used in this exercise.

Can you rewrite your code so that it will work for any file path? For example,
`/path1/path2.2/index.html`